### PR TITLE
Update demo scripts to cord.js:develop

### PR DESCRIPTION
**Pull request by vatsa287**
- Update scripts for new cord.js release `0.9.1-1rc`
- Add asset 
- Update network-score to match ONDC

Usable scripts:
- yarn demo-statement
- yarn demo-asset
- yarn demo-score

### CODE REVIEW

**Review by amarts — APPROVED** [review]

File: src/demo-asset.ts [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+import { createDid } from "./utils/generateDid";
+
+const { NETWORK_ADDRESS, ANCHOR_URI } = process.env;
+
+async function main() {
+  const networkAddress = process.env.NETWORK_ADDRESS ? process.env.NETWORK_ADDRESS : 'ws://127.0.0.1:9944';
+  const anchorUri = ANCHOR_URI ?? '//Alice';
+
+  // Temporarily suppress console.log
+  let originalConsoleLog = console.log;
```
Comment (belyrs) [inline src/demo-asset.ts:14]: Why is this required?

**Review by belyrs — APPROVED** [review]

File: src/demo-asset.ts (hunk 1 above)
Comment (vatsa287) [reply src/demo-asset.ts:14]: AFAIK this is done to not log the outputs in corresponding below methods which might have been added a log for testing or otherwise. 
If no more in use we can remove it but right now does not cause any disruption.
